File: Lab1_python_codes/myFirstLabScript3.py
std_name = input("Name: ")
lab_grade = int(input("Lab: "))
midterm_grade = int(input("Midterm: "))
final_grade = int(input("Final: "))

calculated_lab_grade = lab_grade * 0.25
calculated_midterm_grade = midterm_grade * 0.35
calculated_final_grade = final_grade * 0.40

print(f"Last Score: {calculated_lab_grade + calculated_midterm_grade + calculated_final_grade}")

File: Lab1_cpp_codes/myFirstLabScript3.cpp
#include <iostream>
using namespace std;

int main(){
    cout << "Name: ";
    string name;
    cin >> name;

    cout << "Lab: ";
    int lab_grade;
    cin >> lab_grade;

    cout << "Midterm: ";
    int midterm_grade;
    cin >> midterm_grade;

    cout << "Final: ";
    int final_grade;
    cin >> final_grade;

    double calculated_lab_grade = lab_grade * 0.25;
    double calculated_midterm_grade = midterm_grade * 0.35;
    double calculated_final_grade = final_grade * 0.40;
    
    cout << "Last Score: " << calculated_lab_grade + calculated_midterm_grade + calculated_final_grade << endl;
}